OpenSim/Simulation/SimbodyEngine/Joint.cpp

```diff
@@ -152,6 +152,17 @@ Joint::CoordinateIndex Joint::constructCoordinate(Coordinate::MotionType mt)
     return cix;
 }
 
+Joint::CoordinateIndex Joint::constructCoordinate(Coordinate::MotionType mt,
+                                                  unsigned idx)
+{
+    const auto cix = constructCoordinate(mt);
```

Comment:
Discussed with @aseth1. I'll remove the single-argument version of `constructCoordinate()` and just pass in the loop counter for CustomJoint (since it won't have a `Coord` enum).